tests/integration/test_row_policy/test.py

```diff
@@ -24,7 +24,7 @@ def started_cluster():
         cluster.start()
 
         instance.query('''
-            CREATE DATABASE mydb;
+            CREATE DATABASE mydb ENGINE=Ordinary;
```

Comment:
Is it still possible to create a database without specifying an engine?

Reply:
If `ENGINE` is not specified, default engine corresponding to `default_database_engine` setting will be used. Some tests rely on data paths and blocking `DROP`, so I added `ENGINE=Ordinary`